dev/setup-k3ds

```diff
@@ -32,12 +32,16 @@ k3d cluster create upstream \
   $args
 
 for i in $(seq 1 "$FLEET_E2E_DS_CLUSTER_COUNT"); do
-  k3d cluster create "downstream$i" \
+  cluster="downstream"
```

Comment:
Should we just make it configurable, so we can get rid of `setup-k3d`, which was meant for more "customized" creation of clusters?